### PR TITLE
Feature: ballot giveRightToVote & vote scripts

### PR DESCRIPTION
**Pull request by evchip**
## Description

- Create function `giveRightToVote` to call eponymous method on `ballot` contract
- Create function `vote` to call eponymous method on `ballot` contract
- Create scripts to call `giveRightToVote` and `vote` methods on `ballot` contract
- Install `ethers` as there were some issues with accessing certain methods using the `hardhat` version of the library

## Motivation and Context

To be able to call methods on the deployed Ballot contract

## How Has This Been Tested?

Manually tested calling the methods, which reverted with expected errors due to my account privileges on deployed contract.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Style (style only changes)
- [ ] Refactor (code that does not add new functionality nor fixes a bug)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: scripts/utils/getBallotContract.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  const provider = new ethers.providers.AlchemyProvider("goerli", process.env.ALCHEMY_API_KEY);
+  const signer = new ethers.Wallet(process.env.GOERLI_PRIVATE_KEY, provider);
+
+  const ballotContractFactory = new Ballot__factory(signer);
+  const ballot = ballotContractFactory.attach(contractAddress)
+
+  return ballot
+}
+
+export default getBallotContract
```
Comment (AndyOooh) [inline scripts/utils/getBallotContract.ts:22]: Could we make this a more generic file, eg. `getContract()`?

If we pass `contractName `as we do for the `deploy `command, maybe we could match it against the typechain and factories imports to make the function type and contractFactory types dynamic.

**Review by AndyOooh — CHANGES_REQUESTED** [review]
Suggestion in `getBallotContract`. Otherwise looks good.

File: scripts/utils/getBallotContract.ts (hunk 1 above)
Comment (AndyOooh) [reply scripts/utils/getBallotContract.ts:22]: I think we should just merge this now and we can get back to my suggestion later if relevant.

**Review by AndyOooh — APPROVED** [review]
LGTM

File: scripts/utils/getBallotContract.ts (hunk 1 above)
Comment (evchip) [reply scripts/utils/getBallotContract.ts:22]: I'll look into this refactor if I have time.

**Pull request merged**